js/historian_v2.js

```diff
@@ -382,3 +389,36 @@ historian.HistorianV2.prototype.handleMouse_ = function() {
         this.levelLine_.hideTimeInfo();
       }.bind(this));
 };
+
+/**
+ * Create the selector to support filtering by app feature 
```

Comment:
Creates a selector....

Ideally function header should follow a 3rd person verbs. Creates a, provides ... etc

Reply:
I will fix the comments and formatting in the next commit! Thank you!